src/test/java/org/openrewrite/github/AddCronTriggerTest.java

```diff
@@ -44,8 +49,48 @@ void cronTrigger() {
                   branches:
                     - main
                 schedule:
-                  - cron: "0 18 * * *"
+                  - cron: "%s"
+              """.formatted(cron),
+            spec -> spec.path(".github/workflows/ci.yml")
+          )
+        );
+    }
+
+    static class StaticThreadLocalRandom extends Random{
+        @Override
+        public int nextInt(int any) {
+            return 1;
+        }
+
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+      "@daily,      1 1 * * *",
+      "@weekly,     1 1 * * 1",
+      "@monthly,    1 1 2 * 1",
+      "@hourly,     * 1 * * *",
+      "@yearly,     1 1 2 2 1"
+    })
```

Comment:
This is the best I could think of to test the random generation. These tests are definitely not perfect